pkg/controller/installation/validation.go

```diff
@@ -355,6 +355,9 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 			if *instance.Spec.CalicoNetwork.PolicySetupTimeoutSeconds < 0 {
 				return fmt.Errorf("Installation CNI spec.PolicySetupTimeoutSeconds negative value is not valid")
 			}
+			if instance.Spec.CalicoNetwork.LinuxDataplane != nil && *instance.Spec.CalicoNetwork.LinuxDataplane != operatorv1.LinuxDataplaneIptables {
```

Comment:
I think we probably want this?

```go
if instance.Spec.CalicoNetwork.LinuxDataplane == nil || *instance.Spec.CalicoNetwork.LinuxDataplane != operatorv1.LinuxDataplaneIptables {
```

Basically if the data plane is unset, then we don't know if it's OK to enable this feature or not and we need to wait until it is.

Reply:
Probably also want a check on WindowsDataplane here

Reply:
Similar to the [question](https://github.com/tigera/operator/pull/3186#discussion_r1508020370) I have above, I think this feature would still be desirable in a mixed cluster with windows and linux, if its possible to continue when that is the case. Maybe I could error if _only_ a Windows dataplane is defined, and no Linux one?

Reply:
> Maybe I could error if only a Windows dataplane is defined, and no Linux one?

Is there a reason we haven't implemented this for Windows?